packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -597,12 +695,11 @@ class SemanticsOwner {
     _dirtyNodes.clear();
   }
 
-  SemanticsActionHandler _getSemanticsActionHandlerForId(int id, { @required SemanticsAction action }) {
-    assert(action != null);
```

Comment:
I removed it because I made the argument positional, but I can add it back.  I've moved it to the public API so that if it trips it will be more obvious where the fault lies.